### ISSUE TITLE
Error during run ansible-cmdb after upgrade to 1.30

### ISSUE THREAD

**Issue by proprux** [fboender/ansible-cmdb#192]
Hello,
I've some problem to run ansible-cmdb command after upgrade it to lastest version 1.30.

$ sudo ansible-cmdb --template html_fancy_split -i inventory ../out/
Error parsing: overview.html: No JSON object could be decoded
Traceback (most recent call last):
  File "/bin/../lib/ansiblecmdb/ansible-cmdb.py", line 205, in <module>
    limit=options.limit, debug=options.debug)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/ansible.py", line 57, in __init__
    self._handle_inventory(inventory_path)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/ansible.py", line 109, in _handle_inventory
    self._parse_hosts_inventory(inventory_path)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/ansible.py", line 151, in _parse_hosts_inventory
    hosts_parser = parser.HostsParser(hosts_contents)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/parser.py", line 45, in __init__
    self._apply_section(section, self.hosts)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/parser.py", line 242, in _apply_section
    func(section, hosts)
  File "/usr/lib/python2.7/site-packages/ansiblecmdb/parser.py", line 251, in _apply_section_hosts
    host = hosts[hostname]
KeyError: u'server'

thanks for your help.

ansible version = 2.7.5
python version = 2.7.5
ansible-cmdb version = 1.30

**Comment by fboender** [comment]
Hi Proprux!

Just a guess, but it looks like your inventory is referring to a host called 'server' which doesn't really exist?

I've pushed a change to the master branch which skips hosts that don't actually exists. That should fix your problem. You can test it out by using the master branch, or by waiting for the next release, which I hope to publish soon. I'm going to close this issue for now. Can you reopen it if it turns out this fix didn't help?

**Comment by fplpds-linux** [comment]
Hello

Thanks for your feedback
I made an update and I had the problem again.
After some research I noticed that the errors came from the host of one and the same group, namely the group TEST

I delete the 3 servers from the same group and now everything is working properly.
Thank you for the update of the package. The functionality of the unavailable equipment appearing in red is great !!

thanks again !!
